woocommerce-paypal-payments.php

```diff
@@ -25,7 +25,7 @@
 define( 'PAYPAL_SANDBOX_API_URL', 'https://api.sandbox.paypal.com' );
 define( 'PAYPAL_INTEGRATION_DATE', '2023-03-20' );
 
-define( 'PPCP_FLAG_SUBSCRIPTION', true );
+define( 'PPCP_FLAG_SUBSCRIPTIONS_API', apply_filters( 'ppcp_flag_subscriptions_api', false ) );
```

Comment:
Maybe it can also use an env var as default value instead of `false`? Then you can easily enable it in DDEV etc.

`apply_filters( 'ppcp_flag_subscriptions_api', getenv( 'PPCP_FLAG_SUBSCRIPTIONS_API' ) === '1' )`